roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -6,7 +6,7 @@ dockerConfig:
   execHandlerName: ""
 imageConfig:
   format: {{ openshift.node.registry_url }}
-  latest: false
+  latest: true
```

Comment:
I believe this was meant to be temporary until AE has its own images and its own version settled.